### PR TITLE
Add Contact form

### PR DESCRIPTION
**Pull request by Saritalitu23**
Add form section & Validation to portfolio mobile 
update the code for index.html & style.css.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @Saritalitu23 , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project with an unprofessional readme. Kindly use the microverse template. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Saritalitu23** [comment]
Please be more specific .i am unable understand so i am stuck .

File: styles.css [hunk 1]
```diff
@@ -58,20 +65,625 @@ ul li a {
...
+  background-image: url("asset/background\ img.png");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow: hidden;
+  background-clip: inherit;
+  background-position: 1.875rem -3.1875rem;
+  border-radius: 5rem 0 0 0;
+  position: relative;
+}
```
Comment (AbelG101) [inline styles.css]: - [ ] Ouch! this is very close but it seems like the background image in your design is a bit misaligned as some of the icons in the background image are not in position as shown in the Figma design. As a frontend developer, It's very important you pay attention to small details like this as they give your UI a more polished feel. 😉
- Your design:
  image
- Figma design:
  image

File: styles.css
Comment (AbelG101) [inline styles.css:1]: - [ ] Also pelase make sure you use the given font from the Figma design which is "Poppins" I noticed you forgot to add the proper import statement in your styles file, you just have to copy and paste the below code at the top of your styles file, so they get imported 👇
```css
@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;0,500;0,600;0,900;1,500;1,600&display=swap');
```
- [ ] Also, please make sure all of the texts in your contract section use the "Poppins" font as I noticed the paragraph is using the default browser font.
- Your design:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Saritalitu23 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Form is properly validated ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Saritalitu23,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by skyv26 — COMMENTED** [review]
## Hi @Saritalitu23, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🟨 




## Reason 🗒️ 
You have submitted a project, that is not your repository PR, please submit your repository PR for getting a review. Don't use your partner PR for getting approval.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**